databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -363,11 +376,11 @@ impl From<&broker::Metadata> for proto::Metadata {
             id: metadata.id,
             data_type: proto::DataType::from(metadata.data_type.clone()) as i32,
             entry_type: proto::EntryType::from(metadata.entry_type.clone()) as i32,
-            description: Some(metadata.description.clone()),
-            comment: None,
-            deprecation: None,
-            unit: metadata.unit.clone(),
-            value_restriction: value_restriction_from(metadata),
+            description: metadata.description.clone(),
+            comment: String::new(),
+            deprecation: String::new(),
+            unit: metadata.unit.clone().unwrap_or_default(),
+            value_restrictions: value_restriction_from(metadata),
```

Comment:
if you call this value_restrictions you should call the function the same
```suggestion
            value_restrictions: value_restrictions_from(metadata),
```